<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgConnect</class>
 <widget class="QDialog" name="DlgConnect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to Remote Host</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rms_analyze.qrc">
    <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="rms_analyze.qrc">:/images/mac_connect.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Host Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="HostComboBox"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="icon">
      <iconset resource="rms_analyze.qrc">
       <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="PortLineEdit"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="rms_analyze.qrc">
       <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rms_analyze.qrc"/>
 </resources>
 <connections/>
</ui>
